sandboxed_containers/upgrading-sandboxed-containers.adoc

```diff
@@ -0,0 +1,51 @@
+:_content-type: ASSEMBLY
+[id="upgrading-sandboxed-containers"]
+= Upgrading {sandboxed-containers-first}
+include::_attributes/common-attributes.adoc[]
+:context: upgrading-sandboxed-containers
+
+toc::[]
+
+The upgrade of the {sandboxed-containers-first} components consists of the following three steps:
+
+* Upgrading {product-title} to update the `Kata` runtime and its dependencies.
+
+* Upgrading the {sandboxed-containers-operator} to update the Operator subscription.
+
+* Manually patching the `KataConfig` custom resource (CR) to update the monitor pods.
+
+You can upgrade {product-title} before or after the {sandboxed-containers-operator} upgrade, with the one exception noted below. Always apply the `KataConfig` patch immediately after upgrading {sandboxed-containers-operator}.
+
+[IMPORTANT]
+====
+
+If you are upgrading to {product-title} 4.11 with {sandboxed-containers-first} 1.3, the recommended order is to first upgrade {sandboxed-containers-first} from 1.2 to 1.3, and then upgrade {product-title} from 4.10 to 4.11.
+====
+
+[id="sandboxed-containers-upgrade-resources"]
+== Upgrading the {sandboxed-containers-first} resources
+
+The {sandboxed-containers-first} resources are deployed onto the cluster using {op-system-first} extensions.
+
+//Update the Red Hat CoreOS extension with the updated link once PR is approved for the Understanding section.
+
+The {op-system} extension `sandboxed containers` contains the required components to run Kata Containers such as the Kata containers runtime, the hypervisor QEMU, and other dependencies. You upgrade the extension by upgrading the cluster to a new release of {product-title}.
+
+For instructions on upgrading {product-title}, see xref:../updating/index.adoc#index[Updating Clusters].
```

Comment:
When introducing inline links, please use `For more information about` [1]

[1] https://docs.engineering.redhat.com/display/CCSENABLE/Inline+links